README.md

```diff
@@ -252,4 +268,4 @@ For those with permission to release:
     * Mac users can `brew install gpg pinentry-mac` to get the tools needed.
 * Create a Sonatype [credentials file](http://www.scala-sbt.org/1.0/docs/Using-Sonatype.html#Fourth+-+Adding+credentials).
 * Run `sbt release`
-* Push the newly created tags and version bump commits to `rubicon-project/rubiz`. 
+* `git push` to add the new tags and release commits to master.
```

Comment:
Looks like this got screwed up again.
@skeet70 Did you change this in the README for tut? Maybe Leif should change it on this branch too?